docs/source/modules/vert_coordinate.rst

```diff
@@ -0,0 +1,90 @@
++---------------------+--------------------------------------------------+
+
+Prescribed Levels
+^^^^^^^^^^^^^^^^^
+Sometimes you may wish to manually set the pressure levels that are used in Isca: perhaps you want to directly compare a selection of similar models, or you wish to use the levels provided by another GCM. In order to use this mode we first set ``'vert_coord_option': 'input'``. In addition, we now need to supply a list of the levels we wish to use, which are added using the ``vert_coordinate`` namelist. The namelist ``bk`` contains the list of prescribed sigma levels that the model is to use - an example of which is shown below:
+
+.. code-block:: python
+
+  'vert_coordinate_nml': {
+        'bk': [0.000000, 0.040000, 0.100000, 0.200000, 0.400000, 0.600000, 0.800000, 1.000000],
+        'pk': [0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000],
+       }
+
+Even Sigma
+^^^^^^^^^^
+This option will create an evenly spaced set of levels from sigma level 0 (top) to 1 (surface), scaled by the surface pressure set in ``spectral_dynamics``. To use this scheme, set ``'vert_coord_option': 'even_sigma'`` and also provide a value for ``num_levels``. Increasing the number of levels will decrease the thickness of each model layer.
+
+Uneven Sigma
+^^^^^^^^^^^^
+The use of evenly-spaced model levels is often not the best option for modelling atmospheres, since it can lead to low resolution near the surface and in the troposphere, where the atmosphere is often the most dynamic). In addition, poor resolution near the model top can in certain configurations be unstable, since breaking gravity waves deposit significant amounts of energy into the upper layers. 
+
+An improvement one can make is to use an uneven sigma scheme, where the model levels are not equally spaced, selected with ``'vert_coord_option': 'uneven_sigma'``. In addition to a number of model levels, there are two additional arguments in the namelist that control the spacing of the levels. ``scale_heights`` sets the overall depth of the model, whilst ``surf_res`` changes the distribution of the levels and ``exponent`` controls the bias of these levels towards the surface. When ``surf_res=1`` levels are evenly spaced, and when ``surf_res=0`` model levels follow a function set by the exponent (with mixed behaviour at intermediate values). Setting ``exponent`` to higher values increases the amount of levels in the troposphere.
+
+Hybrid Scheme
+^^^^^^^^^^^^^
+In models with particularly high top levels (for example models including the stratosphere or of other planets), running with a conventional sigma level scheme can be undesirable since the model can become quite unstable; calculations in the uppermost layers are more stable if the layers are flat. To overcome this we can specify a hybrid model level scheme, which combines :math:`\sigma`-levels in the lower atmosphere with :math:`p`-levels in the upper atmosphere (and a transition region in between). 
```

Comment:
Using sigma levels high up 'imprints' the shape of the topography / surface pressure the entire way through the atmosphere, which is undesirable, as you say.